### DISCUSSION TITLE
How the streaming API/Polling/Sleeping Works

### DISCUSSION THREAD

**Discussion by rileymd88** [teslamate-org/teslamate#2246] [General]
Hi there,

Is there a good description of the strategy being used for the streaming API/Polling/Sleeping? I tried searching and found some tidbits but I would like to understand it in more detail.

Thanks,
Riley

**Comment by LelandSindt** [comment]
I _think_ this is what you are looking for. 
https://github.com/adriankumpf/teslamate/discussions/1288#discussioncomment-491506

**Reply by rileymd88** [reply]
Yes this is what I was looking for. Thank you!
